hosts

```diff
@@ -1,24 +1,79 @@
-ansible-node ansible_ssh_host=<Ansible node private IP >
-masternode ansible_ssh_host=<Master-node node private IP >
-workernode0 ansible_ssh_host=<Worker-node node private IP >
-workernode1 ansible_ssh_host=<Worker-node node private IP >
-workernode2 ansible_ssh_host=<Worker-node node private IP >
-workernode3 ansible_ssh_host=<Worker-node node private IP >
+#ansible-node ansible_ssh_host=<Ansible node private IP >
```

Comment:
use host names for HASTE production pipeline (can create multiple inventory files for different contexts)